modules/datastore/src/Service/ResourceProcessor/DictionaryEnforcer.php

```diff
@@ -124,14 +124,25 @@ public function applyDictionary(RootedJsonData $dictionary, string $datastore_ta
    *
    *  {@inheritdoc}
    */
-  public function returnDataDictionaryFields() {
-    // Get DD is mode.
+  public function returnDataDictionaryFields($id = NULL) {
```

Comment:
This function should have a docbloc, the method is unique to this class and is not defined in the interface.

Reply:
Right again